### ISSUE TITLE
Unfortunately this repo got excluded from project

### ISSUE THREAD

**Issue by krishnavpai** [vijaygupta18/Hacktoberfest-2022#536]
Have you tried contacting them?
There was no spam here and I believe all were genuine contributions.
Anyways I don't think you can do much in such cases as its not in your hand.

Eager to learn and contribute more.
Thanks.

**Comment by vijaygupta18** [comment]
> Have you tried contacting them? There was no spam here and I believe all were genuine contributions. Anyways I don't think you can do much in such cases as its not in your hand.
> 
> Eager to learn and contribute more. Thanks.

I tried contacting but there was no response from there side.I wish i could do anything.
Keep learning and contributing.

**Comment by vijaygupta18** [comment]
> So, as I got an email on completing my 4 PRs out of which 2 are from this repo, do i need to create 2 more PRs on another repo now?

yes... I'm not sure but i read in the faq section that you can create PRs on your own repo too...make sure you mark your pr as Hactoberfest accepted.

**Comment by RajputUsman** [comment]
> > So, as I got an email on completing my 4 PRs out of which 2 are from this repo, do i need to create 2 more PRs on another repo now?
> 
> yes... I'm not sure but i read in the faq section that you can create PRs on your own repo too...make sure you mark your pr as Hactoberfest accepted.

How to find out if the desired is repo is excluded from hacktoberfest?

**Comment by manisha069** [comment]
@RajputUsman the best way to find whether the repo is excluded is to check the Issues section of the repo, many maintainers even me created an issue as soon as the repo got excluded, but apart from that many contributors also create issues letting others know that the repo has been excluded (check all issues, open and closed)

If nothing as such is seen then the only way to know is after you have made a PR and at your HacktoberFest Dashboard it shows that the repo was excluded.
